pkg/wrapper/nmt_wrapper_test.go

```diff
@@ -95,7 +95,7 @@ func TestExtendedDataSquare(t *testing.T) {
 	raw := generateRandNamespacedRawData(
 		squareSize*squareSize,
 		appconsts.NamespaceSize,
-		appconsts.SparseShareContentSize,
+		appconsts.SparseShareContentSize+1, // we +1 here to account for the version byte
```

Comment:
```
SparseShareContentSize = ShareSize - NamespaceSize - ShareInfoBytes
```
Doesn't it make sense to increase the `ShareInfoBytes` throughout the repo?

Reply:
[blocking][question]

`SparseShareContentSize` already accounts for the one info byte and I don't think we need to increase it from 1 to 2 so it's not clear to me what the `+1` is for. Any ideas @evan-forbes ?

Reply:
which version? is it this?
```
Where `info (1 byte)` is a byte with the following structure:

- the first 7 bits are reserved for the version information in big endian form (initially, this will be `0000000` for version 0);
- the last bit is a **sequence start indicator**, that is `1` if the share is at the start of a sequence or `0` if it is a continuation share.
```
?

Reply:
yeah, sorry, the info byte. We need to add a byte to replace it

Reply:
Sorry, still don't get the reasoning behind. Now with the new rsmt2d, will we increase the version byte for all protocol? 
If so, then this should be reflected in the `appconsts`, not just the tests.
I am missing some context apparently... Please bare with me :D

Reply:
lol its fine

the old codec handled [][]byte of sizes that aren't powers of two, this one does not, so we just have to make sure that its 512. The current constant is fine and what it should be, we just need to tell the test code to account for the info byte, which was added after the test generation code. 

we don't need to change anything anywhere, only the test code

Reply:
aaah, the leaf size.  Okey, got you :+1: